rfcs/0000-template.md

```diff
@@ -0,0 +1,104 @@
+- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [tc39/test262#0000](https://github.com/tc39/test262/pull/0000)
+
+# Summary
+
+One paragraph explanation of the feature.
+
+# Motivation
+
+Why are we doing this?
+What use cases does it support?
+What is the expected outcome?
+
+# Guide-level explanation
+
+Explain the proposal from the point of view of whichever of test262's
+stakeholders (test writers, test suite consumers, proposal writers,
+test262 maintainers) is going to be affected by it.
+Making lots of use of examples is encouraged.
+
+For RFCs oriented towards the tests and test harness, this section
+should focus on how stakeholders should think about the change,
+and give examples of its concrete impact.
+For policy RFCs, this section should provide an example-driven
+introduction to the policy, and explain its impact in concrete terms.
+
+# Reference-level explanation
+
+This is the technical portion of the RFC.
+Explain the design in sufficient detail that:
+
+- Its interaction with other features is clear.
+- It is reasonably clear how the feature would be implemented.
+- Corner cases are dissected by example.
+
+The section should return to the examples given in the previous section,
+and explain more fully how the detailed proposal makes those examples
+work.
+
+# Drawbacks
+
+Why should we *not* do this?
+
+# Rationale and alternatives
+
+- Why is this design the best in the space of possible designs?
+- What other designs have been considered and what is the rationale for
+  not choosing them?
+- What is the impact of not doing this?
+
+# Prior art
+
+Discuss prior art, both the good and the bad, in relation to this
+proposal.
+A few examples of what this can include are:
+
+- For test harness proposals: Does this feature exist in other test
+  harnesses and what experience have their community had?
+  Is the feature generally considered a net positive?
```

Comment:
i have a mild preference for soft-wrapping instead of hard-wrapping; is there a reason to prefer hard-wrapping?

Reply:
I find soft-wrapped markdown inconvenient on GitHub when viewing diffs, because you often have to scroll side-to-side. Hard-wrapped is also annoying for diffs because you rewrap subsequent lines when you change something, but my standard practice is to limit that by putting a newline after each sentence as well as at the ruler.